### ISSUE TITLE
Shield regeneration, how to keep track in combat?

### ISSUE THREAD

**Issue by Fakaozada** [queryluke/masseffect-5e#330]
Hi guys,

I'm new here, already changed some e-mails w luke and now i'm coming here to discuss further the experiences in my table.

My most recurrent problem in combat: Who gets shields back when the round begins?

It's hard for me to keep track of the shields regenaration, for my PCs and enemies, like, "ok, it's the turn of grunt no. 7, he took damage, but it was last round or this one?"

So I had to start doing some logs of the turns to solve this but it's hard to follow... and kinda annoying...

I had some ideas to fix this for me, like the regeneration consuming an action, so the character have to actively regenarate the shield, but it's not good and the players (of course) didn't like it much...

Other problem that I have with this is the interpratation of when the shields should regenerate, the rule says that when in combat, the shiled regenerates when the pc take the hide or dodge action, but, my players always ask me why they can't regenarate in many other cases, and i'll point out a couple of them:

 - When the character is incapacitated but didn't take any damage for the next round;
 - In battle, after the shield is broken, the pc for some reason (like triyng to encript the data from a terminal while the rest of the group is fighting the enemies forces) is still on the battle, but take no further damage, even if he/she is not using the hide or dodge action.

In these and other cases alike, I'm letting them get their shields back, because I think is reasonable but it got me back to the previous problem, to keep track of it, the pcs are always aware when they can ask if they can regain the shields, but with the npcs aren't...

So, maybe it's more me whinning than having a discussion, but I thought i't would be a good thing to bring up...

**Comment by TheMarmotKing** [comment]
That looks like a really easy fix to me.  I think in combat, allowing some dm discression for when shields Regen is good.     And actually I have an idea for how to make it a bit more lore friendly.  The Dodge action was intended for a game primary based around Melee weapons and ranged weapons slower than bullets.  So I propose the Take Cover action.  Take cover from fire, causing all ranged attack against you to have disadvantage.  I'm not sure if we add a requirement for specific cover types, but I think it would work as a flavored alternative.  

It makes it simple to justify not allowing shields to reboot without taking this action if you think of it as other actions take energy to do.  

As for keeping track, all I can suggest is adding a bit to the character sheet to help.  Treat it like you would temp hp if you are dming
